English Docu and FAQ updated and extended (#194)
greenshot/greenshot: _faqs/2020-04-17-how-can-i-directly-open-an-image-file-in-greenshot.markdown

---
layout: faq
status: publish
published: true

> title: How can I open an image in Greenshot directly?

Reviewer: ```suggestion
title: How can I open an image in Greenshot on double click?
```

Author: imho 'on double click' is not correct, as it is only one option beside 'open with' that is clearly descriped later in the text: "From then, by right clicking on an image and select *Open with* brings Greenshot as an option to choose for open the file."

Reviewer 2: I would almost call it "How can I open an image file in Greenshot"

tags: []
 
---

> Associate an app to a file format by right clicking on an image and select *Open with* from the menu and then *Choose another app*. 

Reviewer: ```suggestion
You can easily open images in Greenshot by using the "Open image from file" and "Open image from clipboard" items from Greenshot's menu. If you also want Greenshot to be the default application for a specific image format (so that it opens e.g. when you double click an image in Windows Explorer) you can proceed as follows:
 
Associate an app to a file format by right clicking on an image and select *Open with* from the menu and then *Choose another app*. 
```

There, on the very bottom of the options list you can add more apps and select Greenshot which is typically located under ```C:\Program Files\Greenshot```.

> From then, by right clicking on an image and select *Open with* brings Greenshot as an option to choose for open the file.

Reviewer: ```suggestion
From then, by right clicking an image and select *Open with* brings Greenshot as an option to choose for open the file.
```
